src/helpers/actions.ts

```diff
@@ -67,22 +68,54 @@ export const clickOnElement = async (
   await element.click();
 };
 
+export const getButton = async (
+  page: DappeteerPage,
+  text: string,
+  options?: {
+    timeout?: number;
+    visible?: boolean;
+  }
+): Promise<DappeteerElementHandle> => {
+  return await Promise.race([
+    getElementByTestId(page, text),
+    getElementByContent(page, text, "button", options),
+  ]);
+};
```

Comment:
That is actually the selector, can you move this code to [selectors.ts](https://github.com/ChainSafe/dappeteer/pull/264/files#diff-394e7acd4f00ee6af3469f403fd30cc351a6d255da834152ff9bc25ac281c0d2)